databricks/koalas/frame.py

```diff
@@ -5458,9 +5458,10 @@ def sort_index(self, axis: int = 0,
         b 1  0  3
         """
         if axis != 0:
-            raise ValueError("No other axes than 0 are supported at the moment")
+            raise NotImplementedError("No other axis than 0 are supported at the moment")
```

Comment:
Should we use `validate_axis` and support `'index'` as well while we are here?

Reply:
yeah, we'd better do that. i'll check if another place to add, too.

thanks for the comment! :)